### PR TITLE
RM-78027 RM-78026 Release over_react 3.7.0

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Included in this release:

__Library Changes:__
- #499 Add PureComponentMixin
    - Allows you to build components similar to [React.PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent)!
- #462, #608 Redux connect performance optimizations
- #501 Add code snippet for Redux connected components

__Analyzer Plugin Changes:__
- #603 Add tests for _"toggle stateful"_ assist
- #604 Restore analyzer plugin Git history

Requested by: @aaronlademann-wf

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/over_react/compare/3.6.0...Workiva:release_over_react_3.7.0
Diff Between Last Tag and New Tag: https://github.com/Workiva/over_react/compare/3.6.0...3.7.0

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1714,5 +1714,505 @@ Initial public release of the library.
...
+[#990]: https://github.com/Workiva/over_react/pull/990
+[#991]: https://github.com/Workiva/over_react/pull/991
+[#992]: https://github.com/Workiva/over_react/pull/992
+[#993]: https://github.com/Workiva/over_react/pull/993
+[#994]: https://github.com/Workiva/over_react/pull/994
+[#995]: https://github.com/Workiva/over_react/pull/995
+[#996]: https://github.com/Workiva/over_react/pull/996
+[#997]: https://github.com/Workiva/over_react/pull/997
+[#998]: https://github.com/Workiva/over_react/pull/998
+[#999]: https://github.com/Workiva/over_react/pull/999
```
Comment (corwinsheahan-wf) [inline CHANGELOG.md:2228]: Oh wow. Should we wait and merge one more PR so we get to 1000 😆

Comment (corwinsheahan-wf) [reply CHANGELOG.md:2228]: Oh nevermind these are just placeholders :(

Comment (aaronlademann-wf) [reply CHANGELOG.md:2228]: lol

**Comment by aaronlademann-wf** [comment]
@Workiva/release-management-pp

**Pull request merged**